flink-runtime/src/main/java/org/apache/flink/runtime/rest/RestClient.java

```diff
@@ -111,7 +112,7 @@
 public class RestClient implements AutoCloseableAsync {
     private static final Logger LOG = LoggerFactory.getLogger(RestClient.class);
 
-    private static final ObjectMapper objectMapper = RestMapperUtils.getStrictObjectMapper();
```

Comment:
Changes to the visibility and `final` keyword in this file do not make sense to me.

Is the `RestClient` common or specific to the runtime?